packages/react-native/src/utils/ensure-node-modules-symlink.ts

```diff
@@ -18,6 +18,21 @@ export function ensureNodeModulesSymlink(
   workspaceRoot: string,
   projectRoot: string
 ): void {
+  const worksapcePackageJsonPath = join(workspaceRoot, 'package.json');
```

Comment:
I was thinking of symlinking `node_modules` folder  itself so we no longer need to maintain individual folders.